### DISCUSSION TITLE
Can multiple docker containers use the same sqlite database?

### DISCUSSION THREAD

**Discussion by TheGorf** [dani-garcia/vaultwarden#4447] [Q&A]
If I have a cluster and a couple containers of vaultwarden running, can I mount /data/ from an NFS source to be used by all of them? I know sqlite supports multiple users at once noting the locking that occurs on write. Obviously the easy change is is to run it off of MariaDB. But, from a simple perspective, does this work? Has anyone run it this way?

**Comment by stefan0xC** [comment]
Not recommended, especially in conjunction with NFS. Cf. https://www.sqlite.org/faq.html#q5
